wfe2/wfe_test.go

```diff
@@ -1856,3 +1853,19 @@ func TestKeyRollover(t *testing.T) {
 		})
 	}
 }
+
+func TestOrder(t *testing.T) {
```

Comment:
Can you add a test case for when `sa.GetOrder` returns an error other than `berrors.NotFoundError`? I think it will be a small change to the mock and one additional test case.

With four test cases I think this test function might also meet the bar to be written in a table-driven style. Thoughts?

Reply:
Done and done.